tiatoolbox/tools/stainextract.py

```diff
@@ -29,23 +29,79 @@
 from tiatoolbox.utils.misc import get_luminosity_tissue_mask
 
 
+def vectors_in_right_direction(e_vectors):
+    """Points the eigen vectors in the right direction.
```

Comment:
Internal functions should be hidden from importing with `_` , but if this prevent test code from using these then ignore my comment.